s/string.🍇

```diff
@@ -56,14 +56,19 @@ string, but those themselves are then immutable again.
   🐊 🔂🐚🔡🍆
   🐊 😛🐚🔡🍆
 
-  📗 Creates a string by copying the memory from the *memory*. 📗
+  📗 Creates a 🔡 by copying the memory from the *memory*. 📗
   ☣️ 🆕 🧠 memory 🧠 🍼 count 🔢 🍇
     count ➡️ size
     🆕🧠🆕 size❗️ ➡️ 🖍bytes
     🚜 bytes 0 memory 0 size❗️
   🍉
 
-  📗 Returns the memory area representing the value of this string. 📗
+  📗
+    Returns the 🧠 storing the value of this 🔡. No copy is performed.
+
+    Do not free the 🧠 manually. Do not use 🧠 after the 🔡 goes out of scope.
```

Comment:
Same here. Maybe we should also mention that this is representation is UTF-8 encoded but I’m not sure whether this should be considered an implementation detail.